### PR TITLE
Add "Jinja2 Language" addon

### PR DESCRIPTION
**Pull request by hongquan**

### CODE REVIEW

**Comment by Defman21** [comment]
I'm not sure the parser will recognize a bitbucket repository. You'll probably have to re-write it to something like that:
```
Jinja2 Language: 
  owner: 
    login: Nguyễn Hồng Quân
  html_url:https://bitbucket.org/hongquan/komodo-jinja
  raw_url: https://bitbucket.org/hongquan/komodo-jinja/downloads/jinja_language-1.0-ko.xpi
  description: short description goes here
  readme: 
    content: >
              long description goes here
  created_at: 2016-07-03
  updated_at: 2016-07-03
```
Also, please edit `languages.yml` so it will appear here: http://komodoide.com/packages/languages/

**Comment by hongquan** [comment]
Thanks, I updated the description.

File: addons.yml [hunk 1]
```diff
@@ -76,31 +76,51 @@ Emmet:
...
-              
+
               ![RemoteDriveTree extension screenshot][1]
-              
+
                  [1]: http://community.activestate.com/files/images/RemoteDriveTree_ScreenShot_v2.png (RemoteDriveTree extension screenshot)
   created_at: 2007-11-16
   updated_at: 2013-03-11
+
+Jinja2 Language:
```
Comment (Defman21) [inline addons.yml]: You need to remove this item from addons because it's already in languages :)

**Comment by hongquan** [comment]
Ok. Removed from  addons.yml.

**Comment by Naatan** [comment]
Great work @hongquan! However this PR is modifying a lot more than just adding your package. Please open a new PR that only performs the task of adding the package, do not modify anything else.

**Comment by hongquan** [comment]
Those additional changes are just "clean whitespace" (it is done automatically by my code editor).

**Comment by Naatan** [comment]
That may be but for a PR we cannot accept this, as it contaminates the VCS history.

**Pull request closed without merging**